blockstore.go

```diff
@@ -97,6 +108,7 @@ func (bs *blockstore) Put(block blocks.Block) error {
 	err = bs.datastore.Put(k, block.RawData())
 	if err == nil {
 		blockCount.Inc()
+		bs.count.Inc()
```

Comment:
I think we can count it once, and use bs.count for metrics too.

Reply:
> I think we can count it once

What do you mean?

>  use bs.count for metrics too

Actually this is a good idea, we can remove the prometheus integration from here, and simply use the counter. Then in TemporalX we can modify the existing blockstore prometheus metric in there to instead of call `AllKeysChan` it just calles `GetTotalCount` ?

Reply:
Yes, that's what I mean

Reply:
ah okay that makes sense ill do that